homeassistant/components/adaptive_lighting/strings.json

```
@@ -0,0 +1,51 @@
+{
+  "title": "Adaptive Lighting",
+  "config": {
+    "step": {
+      "user": {
+        "title": "Choose a name for the Adaptive Lighting",
+        "description": "Every instance can contain multiple lights!",
+        "data": {
+          "name": "Name"
```

```suggestion
          "name": "[%key:common::config_flow::data::name%]"
```

(added in https://github.com/home-assistant/core/pull/40947)

---

Fixed in 6fec1c9, thanks! 👍